arches_lingo/src/arches_lingo/components/generic/ComponentManager/components/ComponentEditor.vue

```diff
@@ -102,9 +119,27 @@ function resetForm() {
     min-height: 0;
 }
 
+.editor-content :deep(.p-formfield) {
+    margin-bottom: 0.65rem;
+}
+
 .editor-content {
-    flex: 1;
-    min-height: 0;
+    min-height: 50vh;
     overflow-y: auto;
+    padding-bottom: 1rem;
+    border-bottom: 0.085rem solid var(--p-menubar-border-color);
+}
+
+.footer {
+    display: flex;
+    justify-content: flex-start;
+    padding: 1rem;
+    border-bottom: 0.085rem solid var(--p-menubar-border-color);
+    background: var(--p-footer-background);
+}
```

Comment:
whatever change here, the footer is no longer acting like a footer. It's not staying at the bottom of the panel.